sql/sql_acl.cc

```diff
@@ -12182,8 +12183,9 @@ int mysql_alter_user(THD* thd, List<LEX_USER> &users_list)
   {
     LEX_USER* lex_user= get_current_user(thd, tmp_lex_user, false);
     if (!lex_user ||
-        replace_user_table(thd, tables.user_table(), lex_user, NO_ACL,
-                           false, false, true))
+        replace_user_table(thd, tables.user_table(), lex_user,
+                           NO_ACL, false,
+                           false, true))
```

Comment:
I prefer less lines of code, not more. I've long noticed that the speed of reading the code is generally measured in "lines per minute" — that is, "less lines" is translated verbatim into the "less time to read".

so, ok, as you prefer. but if I stumble upon this fragment some day and I might fairly automatically reformat it back to occupy one line less.